tests/integration/it_property_resource_test.py

```diff
@@ -0,0 +1,42 @@
+""" Tests for PropertyResource, resource for internationalization based on a
+property file
+
+"""
+import pytest
+
+from timeless.message.property_resource import PropertyResource
+
+
+class TestPropertyResource():
+
+    directory = "resources/messages/"
+
+    @pytest.mark.skip(reason="PropertyResource not implemented yet")
+    def test_get_found(self):
+        """ Test if PropertyResource can return a value that exists"""
+        assert (
+            PropertyResource(
+                 directory=self.directory
```

Comment:
@paulodamaso Does python have ``InputStream``, ``Reader`` or something similar? I think ``PropertyResource`` should work with one of those rather than directly with a File. That way, testing would also become easier, you would pass it a ``ByteArrayInputStream``/``StringReader`` (from a ``String``), instead of having a messages file for test.

Reply:
@amihaiemil It would be right if we were talking about an unite test, but this is an integration test, so I think that we should test this routine accessing the file itself.